evaluation.md

```diff
@@ -5,15 +5,42 @@
+ * However, to get proper binary operator (with the form, "open(..) + blocked(..)", we need to create our own class that mimics Surrounding because we'd need to define the operator inside it.
+ * (^Unless I'm forgetting about something obvious)
+ * Now our tuples might contain Surroundings or OurSurroundings (if users only pass in an 'open', for example, the '+' method will never be called, so the OurSurroundings will never get converted to a Surroundings).
+ * So, we must cast both types of tuples to ShortRule.
+
+As the comments above show, we needed to create a wrapper class OurSurroundings to get the binary operator working, which was something we also didn't expect when writing our ideal example.
+We also realized that Scala requires several parantheses to pass in arguments, so it ends up looking like Lisp or Racket. Because the external DSL uses regular expressions, it was far easier to make get rid of these parantheses. 
+
 **On a scale of 1–10 (where 10 is "a lot"), how much did you have to change your syntax?**
 
+I'd say 3. This is mostly because the DSL we had in mind is supposed to look a lot like a programming language, which seems significantly easier to emulate than implementing, for example, a Picobot language composed entirely of natural language. Most of the "changes" detailed above didn't affect the appearance of our syntax; rather, the "changes" were more focused around implementation details.
+
 **On a scale of 1–10 (where 10 is "very difficult"), how difficult was it to map your syntax to the provided API?**
 
+Probably 7. We felt our syntax mapped pretty well to the provided API in that it retains the same structure (a series of rules where each rule passes in the same parameters). However, abstracting out the start state from the Rule type required writing several additional wrapper functions, as detailed above. 
+
 ## External DSL
 
 _Describe each change from your ideal syntax to the syntax you implemented, and
 describe_ why _you made the change._
 
+The most difficult part of implementing the external DSL was figuring out the proper syntax for the parser.  We initially spent a couple hours struggling to create our own tokens (similar to the wholeNumber built-in token from the JavaTokensParser).  After looking at the sample solution, however, we realized we could just use the RegExParser and not deal with tokens.
+
+Once we had that figured out, implementing the parser was relatively straightforward.  We started from the bottom up, first implementing a parser for RelativeDescription, then for a Surrounding, then ShortRule (our internal representation of a Rule without the start state), etc.  Because we decided to group together states with the same start state (see example code), we had to write some methods that would take bundles of tuples (as an internal data structure) and output Rules, ShortRules, etc.  See ‘descDirSetToSurroundings’ and ‘stateShortsToRules’ from our Semantics.scala.
+
```

Comment:
Good explanation for this change, as it show how you were willing to alter the design to make it better for your goals for the language.